build.rs

```diff
@@ -100,19 +101,43 @@ fn gen_bindings() {
         .expect("Failed writing bindings");
 }
 
-fn build_stub() {
+fn build_extract_stub() {
     const FILTER_STUB: &str = "src/core/filters/packets/bpf/stub.bpf.c";
     let (_, name) = get_paths(FILTER_STUB);
+    let stub_base = format!("{}/{}", env::var("OUT_DIR").unwrap(), name);
+    let stub_out = format!("{}.o", stub_base);
 
     if let Err(e) = SkeletonBuilder::new()
         .source(FILTER_STUB)
-        .obj(format!("{}/{}.o", env::var("OUT_DIR").unwrap(), name))
+        .obj(&stub_out)
         .clang_args(format!("-I{} ", FILTER_INCLUDE_PATH))
         .build()
     {
         panic!("{}", e);
     }
 
+    Command::new("llvm-objcopy")
+        .args([
+            "-O binary",
+            "--set-section-flags .BTF=alloc",
+            "-j .BTF",
+            stub_out.as_str(),
+        ])
+        .arg(format!("{}.BTF", stub_base))
+        .output()
+        .expect("Failed to extract .BTF from stub ELF");
+
+    Command::new("llvm-objcopy")
```

Comment:
Just a note: there's a chance this might not be needed as we probably can generate it starting from the above .BTF